src/main/java/mate/academy/AsyncRequestProcessor.java

```diff
@@ -1,16 +1,32 @@
 package mate.academy;
 
+import java.util.Map;
 import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.Executor;
+import java.util.concurrent.TimeUnit;
 
 public class AsyncRequestProcessor {
+    private static final int SLEEP_TIME = 200;
     private final Executor executor;
+    private final Map<String, UserData> cache = new ConcurrentHashMap<>();
 
     public AsyncRequestProcessor(Executor executor) {
         this.executor = executor;
     }
 
     public CompletableFuture<UserData> processRequest(String userId) {
```

Comment:
The method `processRequest` could benefit from a brief comment explaining its purpose and behavior, especially since it involves asynchronous processing.